h/presenters/group_json.py

```diff
@@ -10,27 +10,42 @@ def __init__(self, group, links_svc=None):
         self.group = group
         self._links_svc = links_svc
 
-    def asdict(self):
-        return self._model(self.group)
+    def asdict(self, expand=[]):
+        model = self._model()
+        self._expand(model, expand)
+        self._inject_urls(model)
+        return model
+
+    def _expand(self, model, expand=[]):
+        if 'organizations' in expand:
+            org_model = {}
```

Comment:
I would expect the organization field to be `null` if the group does not have an organization. This applies in both the expanded and un-expanded cases.

Reply:
What's the advantage of it being null rather than a dict or string? One downside is that if code calls a dict or string method on null it will crash (or if it tries to access `group.organization.name` etc), whereas it'll work on an empty dict or string. http://thecodelesscode.com/case/6 "All nothings are not equal"

All groups are gonna belong to an organization though (as soon as Hannah's DB migrations are done) so we'll be able to remove this case entirely soon

Reply:
If we're concerned with the representation of the sub-resource here which is imminently changing...I'd suggest we omit the property from the response entirely if it's not present for now. i.e. the `organization` property would not be extant on responses at this moment.

`null` isn't ideal, and an empty string (my initial take on the non-expanded version) is admittedly (thanks Rob for making me think about this harder!) not right. Effectively the feature that we're exposing in the API isn't implemented yet, so this is a weird state.